### DISCUSSION TITLE
Using Jupyter Widgets to update Plotly Graphs

### DISCUSSION THREAD

**Discussion by quoc-huynh** [quarto-dev/quarto-cli#5824] [Q&A]
### Description

Hi! This is my first time asking a Question here so I am sorry in advance if my question is not specific enough or has been answered before.  I am currently trying to create a Quarto webpage that is able to get user input and update the plotly graph. I am hoping to have a multiselect drop down menu that based on selection will update the graph with the selected choices. I've tried utilizing Jupyter's widgets but after selection on the webpage, my graph would not update. Specifically, I used this code to try it out: https://plotly.com/python/figurewidget-app/). Are there any recommendations on how to approach this? Thank you so much!

**Comment by mcanouil** [comment]
I don't know enough Jupyter and plotly, but you could problably look into Shiny for Python and the "shinylive" extension.

- https://shiny.rstudio.com/py/docs/shinylive.html
- https://github.com/quarto-ext/shinylive
